content/en/docs/v3.5/tutorials/how-to-get-key-by-prefix.md

```diff
@@ -4,6 +4,31 @@ description: Guide to extracting etcd keys by their prefix
 weight: 300
 ---
 
+## Pre-requisites
+
+* [Install etcdctl](https://etcd.io/docs/v3.5/install/)
```

Comment:
```suggestion
* Install [etcdctl](https://etcd.io/docs/v3.5/install/)
```
nits

Reply:
I think etcdctl is about what is etcdctl. [install etcdctl] is about how to get it.
I don't think we should just highlight the etcdctl.